src/goto-cc/as_mode.cpp

```diff
@@ -303,9 +303,9 @@ int as_modet::as_hybrid_binary(const compilet &compiler)
   std::string saved = output_file + ".goto-cc-saved";
   try
   {
-    file_rename(output_file, saved);
+    std::filesystem::rename(output_file, saved);
```

Comment:
Do we have any tests for this (even manual)? The codecov warning is a bit concerning when refactoring bits like that, which follow some diverging codepaths like exceptions being triggered, etc.

Reply:
I think the only time this is genuinely put to a test at the moment is the "Build Xen with CPROVER tools" CI job. We don't have regression tests for `goto-as` (`as_modet`) at this time.